packages/react-badge/src/components/PresenceBadge/usePresenceBadgeStyles.ts

```diff
@@ -0,0 +1,82 @@
+import { makeStylesCompat, ax } from '@fluentui/react-make-styles';
+import { PresenceBadgeState } from './PresenceBadge.types';
+import { useBadgeStyles } from '../../Badge';
+
+export const usePresenceBadgeRootStyles = makeStylesCompat<PresenceBadgeState>([
+  [
+    null,
+    {
+      padding: 0,
+    },
+  ],
+  [
+    s => s.status === 'busy' || s.status === 'doNotDisturb',
+    theme => ({
+      backgroundColor: theme.alias.color.red.background3,
+      borderColor: theme.alias.color.red.background3,
+    }),
+  ],
+  [
+    s => s.status === 'away',
+    theme => ({
+      backgroundColor: theme.alias.color.marigold.background3,
+      borderColor: theme.alias.color.marigold.background3,
+    }),
+  ],
+  [
+    s => s.status === 'available',
+    theme => ({
+      backgroundColor: theme.alias.color.lightGreen.foreground3,
+      borderColor: theme.alias.color.lightGreen.foreground3,
+    }),
+  ],
+  [
+    s => s.status === 'offline',
+    theme => ({
+      backgroundColor: theme.alias.color.neutral.neutralBackground1,
+      color: theme.alias.color.neutral.neutralForeground3,
+      borderColor: theme.alias.color.neutral.neutralForeground3,
+    }),
+  ],
+  [
+    s => s.status === 'outOfOffice',
+    theme => ({
+      backgroundColor: theme.alias.color.neutral.neutralBackground1,
+      color: theme.alias.color.berry.foreground3,
+      borderColor: theme.alias.color.berry.foreground3,
+    }),
+  ],
+  [
+    s => s.outOfOffice,
+    theme => ({
+      backgroundColor: theme.alias.color.neutral.neutralBackground1,
+    }),
+  ],
+  [
+    s => s.outOfOffice && s.status === 'available',
+    theme => ({
+      color: theme.alias.color.lightGreen.foreground3,
+    }),
+  ],
+  [
+    s => s.outOfOffice && (s.status === 'busy' || s.status === 'doNotDisturb'),
+    theme => ({
+      color: theme.alias.color.red.background3,
+    }),
+  ],
+  [
+    s => s.outOfOffice && s.status === 'away',
+    theme => ({
+      color: theme.alias.color.marigold.background3,
+    }),
+  ],
```

Comment:
I think `available` and `away` should be pink when OOF, right? You could combine `outOfOffice` status with the `outOfOffice` prop for all of those cases:

```suggestion
  [
    s => s.outOfOffice || s.status === 'outOfOffice',
    theme => ({
      backgroundColor: theme.alias.color.neutral.neutralBackground1,
      color: theme.alias.color.berry.foreground3,
      borderColor: theme.alias.color.berry.foreground3,
    }),
  ],
  [
    s => s.outOfOffice && (s.status === 'busy' || s.status === 'doNotDisturb'),
    theme => ({
      color: theme.alias.color.red.background3,
      borderColor: theme.alias.color.red.foreground3,
    }),
  ],
```

Reply:
Not really:

<img width="380" alt="Screen Shot 2021-03-17 at 8 36 37" src="https://user-images.githubusercontent.com/8545105/111461390-ea416680-86fb-11eb-987b-06998f22332a.png">